src/collection-browser.ts

```diff
@@ -1320,6 +1357,71 @@ export class CollectionBrowser
     }
   }
 
+  private async fetchLetterBuckets(
+    type: 'firstTitle' | 'firstCreator'
+  ): Promise<Bucket[]> {
+    if (!this.fullQueryWithoutAlphaFilters) return [];
+
+    const params: SearchParams = {
+      query: this.fullQueryWithoutAlphaFilters,
+      rows: 0,
+      // Only fetch the firstTitle or firstCreator aggregation
+      aggregations: { simpleParams: [type] },
```

Comment:
Question: In reading/review, I was looking for the definition of `simpleParams`. Didn't find it. Low priority.